frontend/src/libs/components/search/search.tsx

```diff
@@ -0,0 +1,21 @@
+import styles from './styles.module.scss';
+
+type Properties = {
+  value: string | number;
+  onValueChange: React.ChangeEventHandler<HTMLInputElement>;
+};
+
+const Search = ({ value, onValueChange }: Properties): JSX.Element => {
+  return (
+    <>
+      <input
```

Comment:
Use react-hook-form here

Reply:
Also use debounce inside of this component, when calling onValueChange

Reply:
@fshabanov so shoud i wrap this componnet into form? And if i use debounce to onValueChange wouldn`t text in input appear after some time?

Reply:
I don't think you need a form here. You can use react-hook-form separately for input field as well.
The text in input will be handled by react-hook-form. It will change as usual. You will call onValueChange using debounce. onValueChange should not be responsible for the input display, it should only get the filtered data (we will send a request to backend with the filter text)

Reply:
I think it's better to wrap input in a form, this will make it more accessible for users

Reply:
done